examples/rapids/02-rapids-dask.ipynb

```diff
@@ -0,0 +1,220 @@
+{
```

Comment:
<p>I think this needs to be <code>SaturnCluster(n_workers=n_workers)</code>.</p><p> </p><p>Otherwise, if the default for <code>dask-saturn</code> changes to, say, <code>2</code>, this <code>wait_for_workers()</code> code would hang forever. I bet that would take us a while to find :grimacing:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/saturncloud/examples/pull/105/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
This won't use the default, it'll use the setting in saturn.json, but it is brittle to define it in both places.

Reply:
What are you suggesting? Not using `wait_for_workers()` at all in these notebooks?

Reply:
Sorry I meant to actually remove n_workers and `wait_for_workers()`. We're settling on defining the cluster spec in the `saturn.json` so we shouldn't pass arguments to `SaturnCluster` at all. I think the experience of spinning up a quickstart should include starting a cluster so workers would already be up

Reply:
Well we could pass it in using an env var, or we could not use `wait_for_workers`

Reply:
final decision: 
- in saturn.json, specify cluster config
- in notebooks, do:
```python
n_workers = 3 # same as specified in saturn.json
cluster = SaturnCluster(n_workers=n_workers)
client = Client(cluster)
client.wait_for_workers(n_workers)
```

Reply:
nice, thanks